src/minidriver/minidriver.c

```diff
@@ -4710,7 +4710,7 @@ DWORD WINAPI CardRSADecrypt(__in PCARD_DATA pCardData,
 		goto err;
 	}
 
-	good = constant_time_eq_i(r, 0);
+	good = constant_time_ge(r, 0);
```

Comment:
https://github.com/OpenSC/OpenSC/issues/3085#issuecomment-2015236209

It was mentioned here, that zero length message does not make much sense so I would probably change this to `r > 0`.